media-gfx/blender/blender-4.4.3-r1.ebuild

```
@@ -0,0 +1,900 @@
+		-DWITH_IMAGE_WEBP="$(usex webp)" # unlisted
+
+		# Audio:
+		# -DWITH_AUDASPACE=OFF
+		# -DWITH_SYSTEM_AUDASPACE=OFF
+		-DWITH_CODEC_FFMPEG="$(usex ffmpeg)"
+		-DWITH_CODEC_SNDFILE="$(usex sndfile)"
+		# -DWITH_COREAUDIO=OFF
+		-DWITH_JACK="$(usex jack)"
+		# -DWITH_JACK_DYNLOAD=
+		-DWITH_OPENAL="$(usex openal)"
+		-DWITH_PIPEWIRE="$(usex pipewire)"
+		# -DWITH_PIPEWIRE_DYNLOAD=
+		-DWITH_PULSEAUDIO="$(usex pulseaudio)"
+		# -DWITH_PULSEAUDIO_DYNLOAD=
+		-DWITH_SDL="$(usex sdl)"
+		# -DWITH_WASAPI=OFF
+
+		# Python:
+		# -DWITH_PYTHON=ON
+		-DWITH_PYTHON_INSTALL="no"
+		-DWITH_PYTHON_INSTALL_NUMPY="no"
+		-DWITH_PYTHON_INSTALL_ZSTANDARD="no"
+		# -DWITH_PYTHON_MODULE="no"
+		-DWITH_PYTHON_SAFETY="OFF"
+		-DWITH_PYTHON_SECURITY="yes"
+		-DPYTHON_INCLUDE_DIR="$(python_get_includedir)"
+		-DPYTHON_LIBRARY="$(python_get_library_path)"
+		-DPYTHON_VERSION="${EPYTHON/python/}"
+		-DWITH_DRACO="yes" # TODO: Package Draco # NOTE use bundled for now
+
+		# Modifiers:
+		-DWITH_MOD_FLUID="$(usex fluid)"
+		# -DWITH_MOD_REMESH=ON
+		-DWITH_MOD_OCEANSIM="$(usex fftw)"
+
+		# Rendering:
+		-DWITH_HYDRA="no" # TODO: Package Hydra
+
+		# Rendering (Cycles):
+		-DWITH_CYCLES_OSL="$(usex osl)"
+		-DWITH_CYCLES_EMBREE="$(usex embree)"
+		-DWITH_CYCLES_PATH_GUIDING="$(usex openpgl)"
+
+		-DWITH_CYCLES_DEVICE_OPTIX="$(usex optix)"
+		-DWITH_CYCLES_DEVICE_CUDA="$(usex cuda)"
+		-DWITH_CYCLES_CUDA_BINARIES="$(usex cuda "$(usex cycles-bin-kernels)")"
+
+		-DWITH_CYCLES_DEVICE_HIP="$(usex hip)"
+		-DWITH_CYCLES_HIP_BINARIES="$(usex hip "$(usex cycles-bin-kernels)")"
+		-DWITH_CYCLES_HYDRA_RENDER_DELEGATE="no" # TODO: package Hydra
+
+		# -DWITH_CYCLES_STANDALONE=OFF
+		# -DWITH_CYCLES_STANDALONE_GUI=OFF
+
+		-DWITH_BLENDER_THUMBNAILER="yes"
+	)
+
+	if has_version ">=dev-python/numpy-2"; then
+		mycmakeargs+=(
+			-DPYTHON_NUMPY_INCLUDE_DIRS="$(python_get_sitedir)/numpy/_core/include"
+			-DPYTHON_NUMPY_PATH="$(python_get_sitedir)/numpy/_core/include"
+		)
+	fi
+
+	# requires dev-vcs/git
+	if [[ "${PV}" == *9999* && "${BVC}" == "alpha" ]]; then
+		mycmakeargs+=(
+			# -DWITH_BUILDINFO="no"
+			-DWITH_EXPERIMENTAL_FEATURES="$(usex experimental)"
+		)
+	else
+		mycmakeargs+=(
+			-DWITH_EXPERIMENTAL_FEATURES="no"
+		)
+	fi
+
+	if use cuda; then
+		# Ease compiling with required gcc similar to cuda_sanitize but for cmake
+		if use cycles-bin-kernels; then
+			local -x CUDAHOSTCXX="$(cuda_gccdir)"
+			local -x CUDAHOSTLD="$(tc-getCXX)"
+
+			if [[ -n "${CUDAARCHS}" ]]; then
+				mycmakeargs+=(
+					-DCYCLES_CUDA_BINARIES_ARCH="$(echo "${CUDAARCHS}" | sed -e 's/^/sm_/g' -e 's/;/;sm_/g')"
+				)
+			fi
+		fi
+	fi
+
+	if use hip; then
+		mycmakeargs+=(
+			-DHIP_ROOT_DIR="$(hipconfig -p)"
+
+			-DHIP_HIPCC_FLAGS="-fcf-protection=none"
+
+			-DCMAKE_HIP_LINK_EXECUTABLE="$(get_llvm_prefix)/bin/clang++"
+
+			-DCYCLES_HIP_BINARIES_ARCH="$(get_amdgpu_flags)"
+		)
+	fi
+
+	if use hiprt; then
+		mycmakeargs+=(
+			-DHIPRT_ROOT_DIR="${ESYSROOT}/usr/lib/hiprt/2.5"
+		)
+	fi
```

```suggestion

		if use hiprt; then
			mycmakeargs+=(
				-DHIPRT_ROOT_DIR="${ESYSROOT}/usr/lib/hiprt/2.5"
				-DHIPRT_COMPILER_PARALLEL_JOBS="$(makeopts_jobs)"
			)
		fi
	fi
```

Move it inside `use hip`, no reason to check for `use hiprt` without hip.
I would set the parallel jobs here, or at least comment it out. Defaults to 1.